src/components/search/FacetsModal.tsx

```diff
@@ -33,17 +34,9 @@ const FacetsModal = (props: FacetsModalProps) => {
       <div className="h-px bg-brand-gray-300 w-screen -mx-4 mb-4 sm:mx-0 sm:w-full">
         {/* Divider */}
       </div>
-      <StandardFacets
-        collapsible={false}
-        showOptionCounts={true}
-        customCssClasses={{
-          standardFacetsContainer: "w-full overflow-y-auto",
-          titleLabel: "text-lg font-bold text-brand-primary",
-          optionLabel: "text-sm",
-          divider: "w-full bg-white my-3",
-        }}
-        searchOnChange={true}
-      />
+      <div className="overflow-y-scroll">
+        <Facets collapsible={false} />
```

Comment:
Should the other options that were on StandardFacets before this PR also be copied over? showOptionCounts, customCssClasses, and searchOnChange

Reply:
Added the options other than `customCssClasses`, people should instead modify the components directly to add their classes.